### PR TITLE
update case stage

### PR DESCRIPTION
**Pull request by songleo**
- move the fallible case to the integration stage
- move the integration case to stable stage
Signed-off-by: Song Song Li <ssli@redhat.com>

### CODE REVIEW

**Comment by clyang82** [comment]
/hold

Let me finish the API changes and then revisit these changes. Thanks.

File: pkg/tests/observability_addon_test.go [hunk 1]
```diff
@@ -208,7 +208,7 @@ var _ = Describe("Observability:", func() {
 			}, EventuallyTimeoutMinute*5, EventuallyIntervalSecond*5).Should(BeTrue())
 		})
 
-		It("[Integration] Modifying managedcluster cr to enable observability", func() {
+		It("[Stable] Modifying managedcluster cr to enable observability", func() {
```
Comment (clyang82) [inline pkg/tests/observability_addon_test.go:211]: Does it stable? I met this case failure in our PR tests 2 days ago.

File: pkg/tests/observability_metricslist_test.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ var _ = Describe("Observability:", func() {
 			testOptions.HubCluster.KubeContext)
 	})
 
-	It("[P2][Sev2][Observability][Stable] Should have metrics which defined in custom metrics allowlist (metricslist/g0)", func() {
+	It("[P2][Sev2][Observability][Integration] Should have metrics which defined in custom metrics allowlist (metricslist/g0)", func() {
```
Comment (clyang82) [inline pkg/tests/observability_metricslist_test.go:32]:
```
INFO[2021-05-31T12:05:18Z] /go/src/github.com/open-cluster-management/observability-e2e-test/pkg/tests/observability_metricslist_test.go:19
INFO[2021-05-31T12:05:18Z]  [P2][Sev2][Observability][Stable] Should have metrics which defined in custom metrics allowlist (metricslist/g0) [It]
INFO[2021-05-31T12:05:18Z]  /go/src/github.com/open-cluster-management/observability-e2e-test/pkg/tests/observability_metricslist_test.go:32
```

File: pkg/tests/observability_addon_test.go (hunk 1 above)
Comment (songleo) [reply pkg/tests/observability_addon_test.go:211]: Got it. let me update this case to Integration.

File: pkg/tests/observability_metricslist_test.go (hunk 2 above)
Comment (clyang82) [reply pkg/tests/observability_metricslist_test.go:32]: ignore this comment ^^. Change to `Integration` is correct.

**Pull request merged**